plugin/minimal/README.md

```diff
@@ -0,0 +1,74 @@
+# minimal
+
+## Name
+
+*minimal* - minimizes the size of the DNS query response 
+
+## Description
+
+The minimal plugin controls the addition of records to the AUTHORITY and ADDITIONAL section of the dns query response.
+
+## Syntax
+
+~~~
+minimal yes | no
```

Comment:
```suggestion
minimal
```

Reply:
Ack these suggestions. They make the README.md look better. Removing the yes | no is a better choice too. Will incorporate them.